### ISSUE TITLE
Change embed to isolate in example

### ISSUE THREAD

**Issue by r12a** [w3c/ttml2#1198]
10.2.12 tts:direction
https://w3c.github.io/ttml2/index.html#style-attribute-direction

> The tts:direction style is illustrated by the following example.
```
<p> The title of the book is "<span tts:unicodeBidi="embed" tts:direction="rtl">نشاط التدويل، W3C</span>" </p>
```

It is recommended to always use isolating bidi controls, rather than embedding, since it solves many problems for RTL scripts.

Therefore we recommend an editorial change to this example, to encourage good usage: change tts:unicodeBidi="embed" to tts:unicodeBidi="isolate".

**Comment by nigelmegitt** [comment]
Since this is editorial I think we could consider this change before transitioning to 2nd Ed PR. @skynavga  what do you think?

**Comment by skynavga** [comment]
@nigelmegitt no, I do not agree; this requires further discussion; furthermore, the recent addition of `isolate` value is not reflected in current TTML/IMSC implementations;

**Comment by r12a** [comment]
It's an example. I wouldn't have thought this change would be problematic, since the spec allows one to do this. One would hope that changing the example would have the effect of encouraging people to implement support for that value, if nothing else.  It's certainly desirable to isolate spans in bidrectional text.

**Comment by skynavga** [comment]
@r12a my primary concern is that the addition of `isolate` is a very recent change with generally unknown implementation support; the current value is not technically incorrect, so I would prefer leaving it in place for the time being, and we can revisit this issue at a future point without prejudice;

**Comment by r12a** [comment]
I still think you should be encouraging good practice rather than bad in examples.
